docs/handlers/overview.md

```diff
@@ -42,7 +40,7 @@ See the documentation for
 [`BaseRenderer`][mkdocstrings.handlers.base.BaseRenderer].
 
 Check out how the
-[Python handler](https://github.com/pawamoy/mkdocstrings/blob/master/src/mkdocstrings/handlers/python.py)
+[Python handler](https://github.com/mkdocstrings/python/blob/master/src/mkdocstrings_handlers/python)
```

Comment:
https://github.com/mkdocstrings/mkdocstrings/pull/356 will remove this destination though

Reply:
Hmmm no I think it's right: this link points to the extracted Python handler, i.e. its own repository in the org, and not mkdocstrings' repo.